### DISCUSSION TITLE
How can I use the updated version of repository?

### DISCUSSION THREAD

**Discussion by EikoSAIJOU334** [nextflow-io/nextflow#2008] [Q&A]
Hi.

I have been analyzing my NGS data with Nextflow / Github / Docker.
Recently, the Nextflow pipeline I used is updated.

The alert message is bellow:
NOTE: Your local project version looks outdated - a different revision is available in the remote repository [(Hexadecimal digits)]

I would like to use the new version, but I couldn't.
How can I specify the latest version?

Thanks,

**Comment by EikoSAIJOU334** [comment]
My problem may be the same as for issue #1988.
My Nextflow ver. is 20.10.0, and I couldn't specify the GitHub tag.
I will try the new Nextflow version 21.03.0.

**Reply by pditommaso** [reply]
To update a pipeline already downloaded use the `pull` command https://www.nextflow.io/docs/latest/cli.html#pull

**Comment by EikoSAIJOU334** [comment]
Thanks, Paolo!
 I already tried the pull command, but it didn’t work at least my interested repository.
 (and also test the  -r option when I run, but it didn’t work) 
Anyway, I’ll contact the owner of the repository. Thank you so much.
Eiko
